phase2/kubeadm/do

```diff
@@ -18,13 +18,14 @@ upgrade_master() {
   update_host_kubeadm ${MASTER}
   validate
 
-  INIT_PARAM=$(execute_host ${MASTER} "sudo cat ${KUBEADM_INIT_PARAM_FILE}")
+  execute_host ${MASTER} "sudo sed -i \"s|^kubernetesVersion: .*|kubernetesVersion: \\\"${KUBERNETES_VERSION}\\\"|\" ${KUBEADM_CONFIG_FILE}"
+
   case "${UPGRADE_METHOD}" in
     "init")
-      execute_host ${MASTER} "sudo kubeadm init --skip-preflight-checks --kubernetes-version ${KUBERNETES_VERSION} ${INIT_PARAM}"
+      execute_host ${MASTER} "sudo kubeadm init --skip-preflight-checks --config ${KUBEADM_CONFIG_FILE}"
```

Comment:
My understanding is this will not work. The init upgrade path is used for 1.6 -> 1.7. I believe init with config did not exist in 1.7. That is why I went with specifying params.